bucket/lightbulb.json

```diff
@@ -1,29 +1,26 @@
 {
-    "version": "1.6.4.1",
-    "description": "Continuously adjusts gamma, transitioning the display color temperature from cold blue to warm yellow.",
+    "version": "2.3",
+    "description": "Computer screen gamma adjuster based on the current time",
     "homepage": "https://github.com/Tyrrrz/LightBulb",
     "license": "GPL-3.0-or-later",
-    "notes": "If the gamma doesn't change under 6600K, run '$dir\\Manual_Gamma_Registry_Fix.reg'.",
-    "url": "https://github.com/Tyrrrz/LightBulb/releases/download/1.6.4.1/LightBulb-Portable.zip",
-    "hash": "15ba9a6a9c890da8a787dc7af041349b66e179821ab7a28dc115f80c458853c4",
-    "pre_install": "if (!(Test-Path \"$persist_dir\\Configuration.dat\")) { Set-Content \"$dir\\Configuration.dat\" -Value '{}' -Encoding Ascii | Out-Null }",
+    "url": "https://github.com/Tyrrrz/LightBulb/releases/download/2.3/LightBulb.zip",
+    "hash": "402cbd6e206713ea13ca3097acaa9d36b96f200236eb70bd3deb8bd26ba371b2",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\Settings.json\")) { Set-Content \"$dir\\Settings.json\" -Value '{\"IsAutoUpdateEnabled\": false}' -Encoding Ascii }",
     "bin": [
         [
-            "LightBulb_Portable.bat",
+            "LightBulb.exe",
             "LightBulb"
         ]
     ],
     "shortcuts": [
         [
-            "LightBulb_Portable.bat",
-            "LightBulb",
-            "",
-            "LightBulb.exe"
+            "LightBulb.exe",
+            "LightBulb"
         ]
     ],
-    "persist": "Configuration.dat",
+    "persist": "Settings.json",
```

Comment:
Do migration or at least notify users, that you are going to throw all of theirs configurations

Reply:
Though the old Configuration.dat is also in json format, all items have different names. So I only add a notification.